portal/src/graphql/adminapi/UserDetailsScreen.tsx

```diff
@@ -424,10 +425,11 @@ const UserDetails: React.VFC<UserDetailsProps> = function UserDetails(
           />
         </PivotItem>
         <PivotItem
+          className={"h-full pt-8"}
```

Comment:
`height: x%` only works if the container has a explicitly specified height.
https://developer.mozilla.org/en-US/docs/Web/CSS/height#formal_definition

The container of this element doesn't have a explicitly specified height, so prefer not to use `h-full` here.

Same in L453 in this file.

Reply:
a bit doubt on this statement `If the height of the containing block is not specified explicitly (i.e., it depends on content height)`. As the parent container of the element is a flex-child using `flex-1`, seems it's not depending on content height?

Reply:
I think "it depends on content height" is only one case. Height of `flex-1` is affected by the remaining spaces of the container, so I don't think it is "specified explicitly".
In fact, some old browsers such as safari 13 / 14 will not work with this style if I remember correctly (I seems working in the latest versions tho)

Reply:
I've searched a bit on this issue, and found some related discussions:
https://stackoverflow.com/a/61942083
https://stackoverflow.com/questions/33636796/chrome-safari-not-filling-100-height-of-flex-parent

And my understanding is:
- Originally, height: x% cannot be resolved correrctly if the container doesn't specify the height explicitly (height: 100px or something like that)
- But now, modern browsers start accepting `height: 100%` under a certain conditions (Such as using `align-items: stretch`).
- An always working solution is still using nested flex containers.